opacus/grad_sample/grad_sample_module.py

```diff
@@ -282,7 +282,7 @@ def capture_activations_hook(
 
         if not hasattr(module, "activations"):
             module.activations = []
-        module.activations.append(forward_input[0].detach())  # pyre-ignore
+        module.activations.append([t.detach() for t in forward_input])  # pyre-ignore
```

Comment:
should this be append or extend?

Reply:
I think append is right, as activations is a list of tuples in this case